collaboration/README.md

```diff
@@ -1,5 +1,15 @@
 # Collaboration
 
-<!-- group norms summary -->
+## Group Norms
+
+Our group is committed to fostering a respectful and supportive environment where we honor each other's time and cultural differences, while providing flexible communication channels to accommodate diverse schedules.
```

Comment:
the summary should be under 'collaboration' and the norms under the 'group norms' header